components/brave_ads/browser/notification_helper_win.cc

```diff
@@ -27,4 +113,129 @@ NotificationHelper* NotificationHelper::GetInstance() {
   return NotificationHelperWin::GetInstance();
 }
 
+///////////////////////////////////////////////////////////////////////////////
+
+bool NotificationHelperWin::IsFocusAssistEnabled() const {
+  const auto nt_query_wnf_stated_data_func =
+      GetProcAddress(GetModuleHandle(L"ntdll"), "NtQueryWnfStateData");
+
+  const auto nt_query_wnf_stated_data_ptr =
```

Comment:
I believe this should be `const auto*`, but not sure why the google checks in the compiler aren't complaining about it

Reply:
I have refactored the code and removed ptr as changing to const auto * caused a compiler error